src/OrbitGl/CaptureViewElement.h

```diff
@@ -48,6 +54,10 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   void OnDrag(int x, int y) override;
   [[nodiscard]] bool Draggable() override { return true; }
 
+  [[nodiscard]] bool IsMouseOver(const Vec2& mouse_pos);
+  [[nodiscard]] bool HandleMouseWheelEvent(const Vec2& mouse_pos, int delta,
```

Comment:
I'm always confused by what the returned "bool" means in this context. Could we have an enum maybe? Or at least a comment?

Reply:
That's a good idea, I've changed this to return an enum so the intent is a lot clearer.